deploy/percona-broker.yaml

```diff
@@ -0,0 +1,557 @@
+---
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: percona-service-broker
+---
+apiVersion: v1
+kind: Service
+metadata:
+ name: percona-service-broker
+ namespace: percona-service-broker
+spec:
+ selector:
+   app: percona-service-broker
+ ports:
+ - protocol: TCP
+   port: 8081
+   targetPort: 8081
+---
+apiVersion: servicecatalog.k8s.io/v1beta1
+kind: ClusterServiceBroker
+metadata:
+  name: percona-service-broker
+  namespace: percona-service-broker
+spec:
+  url: 'http://percona-service-broker.percona-service-broker.svc:8081'
+---
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: percona-service-broker
+  namespace: percona-service-broker
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRoleBinding
+metadata:
+  name: percona-service-broker
+  namespace: percona-service-broker
+roleRef:
+  apiGroup: rbac.authorization.k8s.io
+  kind: ClusterRole
+  name: percona
+subjects:
+  - kind: ServiceAccount
+    name: percona-service-broker
+    namespace: percona-service-broker
+---
+kind: ClusterRole
+apiVersion: rbac.authorization.k8s.io/v1beta1
+metadata:
+  name: percona
+  namespace: percona-service-broker
+rules:
+- apiGroups:
+  - pxc.percona.com
+  resources:
+  - perconaxtradbclusters
+  - perconaxtradbclusters/status
+  - perconaxtradbclusterbackups
+  - perconaxtradbclusterbackups/status
+  - perconaxtradbclusterrestores
+  - perconaxtradbclusterrestores/status
+  verbs:
+  - get
+  - list
+  - watch
+  - create
+  - update
+  - patch
+  - delete
+- apiGroups:
+  - psmdb.percona.com
+  resources:
+  - perconaservermongodbs
+  - perconaservermongodbs/status
+  - perconaservermongodbbackups
+  - perconaservermongodbbackups/status
+  - perconaservermongodbrestores
+  - perconaservermongodbrestores/status
+  verbs:
+  - get
+  - list
+  - update
+  - watch
+  - create
+- apiGroups:
+  - apps
+  resources:
+  - deployments
+  - replicasets
```

Comment:
is broker creates 'replicasets' or 'statefulsets' or 'jobs' or 'cronjobs' or 'certmanager' ?

Reply:
here is what we get from Kubernetes when psmdb or pxc roles has been applied


```
Error from server (Forbidden): error when creating "psmdbrole.yaml": roles.rbac.authorization.k8s.io "percona-server-mongodb-operator" is forbidden: user "system:serviceaccount:percona-service-broker-3:percona-service-broker" (groups=["system:serviceaccounts" "system:serviceaccounts:percona-service-broker-3" "system:authenticated"]) is attempting to grant RBAC permissions not currently held:
{APIGroups:[""], Resources:["persistentvolumeclaims"], Verbs:["get" "list" "watch" "create" "update" "patch" "delete"]}
{APIGroups:[""], Resources:["pods"], Verbs:["get" "list" "watch" "create" "update" "patch" "delete"]}
{APIGroups:[""], Resources:["pods/exec"], Verbs:["get" "list" "watch" "create" "update" "patch" "delete"]}
{APIGroups:[""], Resources:["secrets"], Verbs:["get" "list" "watch" "create" "update" "patch" "delete"]}
{APIGroups:[""], Resources:["services"], Verbs:["get" "list" "watch" "create" "update" "patch" "delete"]}
{APIGroups:["batch"], Resources:["cronjobs"], Verbs:["get" "list" "watch" "create" "update" "patch" "delete"]}
{APIGroups:["certmanager.k8s.io"], Resources:["certificates"], Verbs:["get" "list" "watch" "create" "update" "patch" "delete" "deletecollection"]}
{APIGroups:["certmanager.k8s.io"], Resources:["issuers"], Verbs:["get" "list" "watch" "create" "update" "patch" "delete" "deletecollection"]}
{APIGroups:["policy"], Resources:["poddisruptionbudgets"], Verbs:["get" "list" "watch" "create" "update" "patch" "delete"]}
```

ClusterRole shall contain all those permissions for inheritance